lib/ansible/executor/module_common.py

```diff
@@ -92,7 +92,22 @@
 # LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import os
+import os.path
 import sys
+import __main__
+
+# With Python3 on Ubuntu, we pick up this script in the temporary directory.
```

Comment:
We don't know.  We do know that it happens with both Ubuntu15 (python-3.4) and Ubuntu16 (python-3.5) and that it does not happen with Fedora 23 (python-3.4).